api-test/fetch-user.js

```diff
@@ -0,0 +1,27 @@
+
+const cardContainer = document.getElementById('card-container');
+const fetchData = async()=>{
+  try{
+    const res = await fetch('https://jsonplaceholder.typicode.com/users')
+    const userData= await res.json();
+    console.log(userData);
```

Comment:
console.log is never supposed to be committed, In production code you never use console.log instead logger is used which you will learn later

Reply:
Sir I've removed the console.log